package.json

```diff
@@ -39,7 +39,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "chalk": "^2.3.0",
-    "clang-format": "^1.0.53",
+    "clang-format": "1.2.1",
```

Comment:
Why no `^`?

Reply:
`clang-format` doesn't follow semver. 1.0.53 and 1.0.55 for example don't output the same string. By fixing it we can prevent inconsistencies between installations made at different times or different versions of node.